neuromation/api/abc.py

```diff
@@ -99,12 +99,30 @@ class ImageProgressPush:
     dst: RemoteImage
 
 
+@dataclass(frozen=True)
+class ImageProgressSave:
+    job: str
+    dst: RemoteImage
+
+
 @dataclass(frozen=True)
 class ImageProgressStep:
     message: str
     layer_id: str
 
 
+@dataclass(frozen=True)
+class ImageCommitStarted:
+    job_id: str
+    container: str
```

Comment:
What `container` string is?
In jobs API we have complex `Container` dataclass. I have an understanding of what dataclass fields mean. We'll have documentation for it soon.
As a user, I have no idea what to do with container-as-string except printing it may be (but what is the string format)?

Reply:
it's 64-long SHA256 hash of docker container

Reply:
mostly technical information, but since we reveal operations "commit" and "push", I decided to print it as well.

Reply:
Hash of docker container or image?
If this is a local docker container hash -- how can I use this info as a client?
Client has no access to docker daemon deployed on the node.

Reply:
hash of docker container. 

> Client has no access to docker daemon deployed on the node.

 true. I will remove `container: str`.

Reply:
In this case, we also should not print `Pushing image artemyushkovskiy/test:latest => image://artemyushkovskiy/test:latest` because the source image does not  make sense for the user

Reply:
I removed messages about "container", but I will not fix docker-push messages `"Pushing image artemyushkovskiy/test:latest => image://artemyushkovskiy/test:latest"` in this PR as it is a very minor issue that requires lots of changes.